src/scripts/components/Play.js

```diff
@@ -426,6 +426,7 @@ class Play extends Component<Props, State> {
 
     // Space key
     if (e.keyCode === 32) {
+      e.preventDefault()
```

Comment:
This prevents scrolling when the user presses "space" in an attempt to play/pause the video